reporter-web-app/src/components/AboutModal.js

```diff
@@ -167,6 +181,17 @@ const AboutModal = (props) => {
                         third-party software components under other open source licenses.
                         See OSS Review Toolkit code repository for further details.
                     </p>
+                    {
+                        !!creationDate
+                        && (
+                            <p>
+                                This ORT run started on
```

Comment:
https://github.com/oss-review-toolkit/ort/issues/3727 "In order to find out easily, while studying a report, when it was created, I would like to have the creation date present in the report." 

I presumed this was the WebApp report creation date itself not the Analyzer start time.`
It may occur that the analyzer, scanner and reporter were run on different dates so I get why you implemented the analyzer time but then should change the text here to "This ORT report is based on analysis started on" 
Grammar-wise using "This ORT run" is weird as this sentence is part of a static HTML file so nothing is "running"

Reply:
Yes you're right, but I changed your suggestion to `This ORT report is based on an analysis started on`.

Reply:
But to me the question remains: Why pick the analyzer time instead of, say, the scanner time? I cannot currently think of a good justification for using another tool's time than the time the report was generated. That is, if we want to stick to single time. Another option of course would be to simply include the time (and duration, maybe) of *all* tools that were run.

Reply:
If we simply add the time the report was created, it would not reflect the time the data in the report was collected, which is ultimately what matters, in my opinion.
We could add the start time of the scanner as well, since data is collected there too.

Reply:
> We could add the start time of the scanner as well, since data is collected there too.

And the advisor, and the evaluator, and ... you see where this is going. Hence my proposal to stick to the report generation time.

Reply:
I do see where this is going. We had this discussion during implementation. 
But as I say, I feel like the time the report was created, is the most uninteresting of all the times.

Reply:
I'm not saying it needs to happen in this change, but why not go one step further and add the start and end times for all tools? Then the report could also show the durations which would be useful information.

Reply:
> why not go one step further and add the start and end times for all tools?

Quoting myself from [above](https://github.com/oss-review-toolkit/ort/pull/3978#discussion_r626432039):

> Another option of course would be to simply include the time (and duration, maybe) of all tools that were run.

😉

Reply:
See, I support that 🙂

Reply:
@mnonnenmacher @sschuberth, do you want these multiple values to be added in scope of this PR? (I'd approve otherwise as it LGTM)

Reply:
It's ok from my side, we can improve this later.